ckanext/dalrrd_emc_dcpr/blueprints/error_report.py

```diff
@@ -0,0 +1,443 @@
+import json
+import logging
+import typing
+from functools import partial
+import ckan.lib.navl.dictization_functions as dict_fns
+import ckan.lib.helpers as h
+import ckan.model
+from flask import Blueprint, redirect, request
+from flask.views import MethodView
+from ckan.views.home import CACHE_PARAMETERS
+from ckan.views.dataset import url_with_params
+from ckan.plugins import toolkit
+from ckan.logic import clean_dict, parse_params, tuplize_dict
+
+from ..helpers import get_status_labels
+
+logger = logging.getLogger(__name__)
+
+error_report_blueprint = Blueprint(
+    "error_report", __name__, template_folder="templates", url_prefix="/error_report"
+)
+
+
+@error_report_blueprint.route("/")
+def get_error_reports():
+    return _get_error_reports_list("error_report_list_public", True)
+
+
+@error_report_blueprint.route("/rejected_error_reports")
+def get_error_reports_list():
+    return _get_error_reports_list("error_report_list_public")
```

Comment:
this function uses the same action as the previous with the difference being the  should_show_create_action, why it has the end point name "rejected_error_reports", i expect something as a new list of rejected error reports

Reply:
you are right @Mohab25, this needs an update.